src/NServiceBus.AcceptanceTests/EndpointTemplates/ConfigureExtensions.cs

```diff
@@ -6,16 +6,14 @@
 
     public static class ConfigureExtensions
     {
-        public static async Task DefineTransport(this EndpointConfiguration config, RunDescriptor runDescriptor, EndpointCustomizationConfiguration endpointCustomizationConfiguration)
+        public static async Task DefineTransport(this EndpointConfiguration config, IConfigureEndpointTestExecution transportConfiguration,RunDescriptor runDescriptor, EndpointCustomizationConfiguration endpointCustomizationConfiguration)
```

Comment:
Could this break downstreams that have created their own acceptance tests in addition to the ones we ship from core?

Reply:
potentially yes as we're not shipping the DefaultServer afaik. I guess we can provide both options to avoid an unnecessary breaking change.

Reply:
add the old API back.